### ISSUE TITLE
Data without barcode

### ISSUE THREAD

**Issue by edusoto** [jbloomlab/dms_tools2#8]
Hi,

I was wondering if it is possible to analyze amplicon reads from deep mutational scanning data even if the reads do not have UMI (barcode) using dms_tools suite?
Thanks in advance

Eduardo

**Comment by jbloom** [comment]
Yes, it may be possible, although it somewhat depends on what you are trying to accomplish. Much of `dms_tools2` is tailored for specific experimental designs (such as barcoded subamplicons), so depending on what you are trying to accomplish it might be more effective to write a custom Python script.

For instance, you might want to create a counts file similar to that generated by `dms2_bcsubamp` (https://jbloomlab.github.io/dms_tools2/dms2_bcsubamp.html#counts-file) and then just use `dms_tools2` for downstream steps? But I don't think that `dms2_bcsubamp` itself will work for non-barcoded reads.

If you want to post a bit more specifics of what you are trying to accomplish I can try to offer more specific help.
